src/_rules/grid.js

```diff
@@ -37,7 +41,7 @@ export const grid = [
       bounds.gridCol,
       numericHandler
     ),
-    { autocomplete: ['(row|col)-span-<num>'] }
+    { autocomplete: ['(row|col)-span-<num>'] },
```

Comment:
Please turn off whatever lint/prettier is doing this (altering/fixing dangling-commas and syntax) - we definitely should configure a linter so we get consistent style, but for now this just makes PRs harder to read :)

Reply:
alright. It is my VScode prettier. turning this off now